shell/platform/darwin/macos/BUILD.gn

```diff
@@ -82,7 +82,10 @@ source_set("flutter_framework_source") {
 
   public_configs = [ "//flutter:config" ]
 
-  defines = [ "FLUTTER_FRAMEWORK" ]
+  defines = [
+    "FLUTTER_FRAMEWORK",
+    "FLUTTER_ENGINE_NO_PROTOTYPES",
```

Comment:
Looks like this breaks FlutterEngine.mm due to a few symbols defined in the `#infndef FLUTTER_ENGINE_NO_PROTOTYPES` block:
* `FlutterEngineCollectAOTData`
* `FlutterEngineRunsAOTCompiledDartCode`
* `FlutterEngineCreateAOTData`

Reply:
I apparently didn't rebuild locally after rebasing on master 😳 The code that's not compiling is new, so hadn't been updated to use the proc table.